Description: Remove versioned dep to stop test suite using pypi
Author: Mark Hymers <mark.hymers@pexip.com>

--- billiard-3.6.0.0.orig/requirements/test.txt
+++ billiard-3.6.0.0/requirements/test.txt
@@ -1,3 +1,3 @@
 case>=1.3.1
-pytest>=3.0
+pytest
 psutil
